en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/adding-debug-logs.md

```diff
@@ -1,126 +1,33 @@
 
-Configure the logging in the `log_config.toml` in the directory `<CHOREO-CONNECT_HOME>/docker-compose/<choreo-connect>/conf/`.
+If you need debug logs to be only enabled in package level, set the log level as `DEBG` only in the relavant [package level configuration]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/configure-logs-adapter/#adapter-package-level-configurations).
 
-The following log levels are supported at root and package level. 
-"INFO", "WARN", "DEBG", "FATL", "ERRO", "PANC"
+### Enforcer
 
-Additionally, log rotation can be configured as follows for adapter logs.
+Configure `log4j2.properties` file and make the value of `rootLogger.level` as `DEBUG` will enable debug logs as well as access logs in enforcer. Refer [enforcer log configurations]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/configure-logs-enforcer/#enforcer-log-configurations) for more details.
 
-```toml
-## Adapter root Level configurations
+!!! Note
+    If you want to enable debug logs and access logs seperately, you need to define a new logger. For more information refer to [Configuring Log4j2 Loggers]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/configure-logs-enforcer/#configuring-log4j2-loggers).
 
-logfile = "logs/adapter.log" # This file will be created inside adapter container.
-logLevel = "INFO"
-
-[rotation]
-MaxSize = 10 # In MegaBytes (MB)
-MaxBackups = 3
-MaxAge =  2   # In days
-Compress = true
-
-## Adapter package Level configurations
-
-[[pkg]]
-name = "github.com/wso2/adapter/internal/adapter"
-logLevel = "INFO"
-
-[[pkg]]
-name = "github.com/wso2/adapter/internal/oasparser"
-logLevel = "INFO"
-```
-
-## Enforcer
-
-### Enable debug logs
-
-Configure `log4j2.properties` located in the directory `<CHOREO-CONNECT_HOME>/docker-compose/<choreo-connect>/conf/`.
-Make relevant packages to `DEBUG` level.
-
-```yaml
-logger.enforcer.level = DEBUG
-```
-
-```yaml
-appender.ENFORCER_LOGFILE.fileName = logs/enforcer.log
-appender.ENFORCER_LOGFILE.filePattern = /logs/enforcer-%d{MM-dd-yyyy}.log
-appender.ENFORCER_LOGFILE.layout.type = PatternLayout
-appender.ENFORCER_LOGFILE.layout.pattern = [%d][%X{traceId}] %5p - %m%ex%n
-...
-```
-
-### Enable access logs
-
-Configure `log4j2.properties` located in the directory `<CHOREO-CONNECT_HOME>/docker-compose/<choreo-connect>/conf/`.
-
-By making the value `logger.mgw-enforcer-interceptors.level` to `DEBUG` in the below configuration will enable access logs without restarting the enforcer.
+The access log format will be as follows. It will print the server time, trace Id from envoy, gRPC service method, gRPC status code, response time according to the above configuration.
```

Comment:
```suggestion
The access log format will be as follows. It will print the server time, trace Id from Envoy, gRPC service method, gRPC status code, and response time according to the above configuration.
```